
2022-01-25 11:34:17,700:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 22, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 2102, in convert
    ) in read_images(rawdata, kwargs["colorspace"], kwargs["first_frame_only"]):
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1483, in read_images
    imgdata, imgformat, default_dpi, colorspace, rawdata
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1189, in get_imgmetadata
    raise AlphaChannelError("Refusing to work on images with alpha channel")
img2pdf.AlphaChannelError: Refusing to work on images with alpha channel

2022-01-25 11:41:55,402:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 22, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 2102, in convert
    ) in read_images(rawdata, kwargs["colorspace"], kwargs["first_frame_only"]):
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1483, in read_images
    imgdata, imgformat, default_dpi, colorspace, rawdata
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1189, in get_imgmetadata
    raise AlphaChannelError("Refusing to work on images with alpha channel")
img2pdf.AlphaChannelError: Refusing to work on images with alpha channel

2022-01-25 11:52:02,976:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 22, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 2102, in convert
    ) in read_images(rawdata, kwargs["colorspace"], kwargs["first_frame_only"]):
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1483, in read_images
    imgdata, imgformat, default_dpi, colorspace, rawdata
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1189, in get_imgmetadata
    raise AlphaChannelError("Refusing to work on images with alpha channel")
img2pdf.AlphaChannelError: Refusing to work on images with alpha channel

2022-01-25 11:56:23,654:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 22, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/PIL/Image.py", line 546, in __getattr__
    raise AttributeError(name)
AttributeError: filename

2022-01-25 12:09:23,764:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 25, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/PIL/Image.py", line 546, in __getattr__
    raise AttributeError(name)
AttributeError: filename

2022-01-25 12:10:01,965:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 25, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/PIL/Image.py", line 546, in __getattr__
    raise AttributeError(name)
AttributeError: filename

2022-01-25 12:13:02,597:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 43, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 2102, in convert
    ) in read_images(rawdata, kwargs["colorspace"], kwargs["first_frame_only"]):
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1483, in read_images
    imgdata, imgformat, default_dpi, colorspace, rawdata
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1189, in get_imgmetadata
    raise AlphaChannelError("Refusing to work on images with alpha channel")
img2pdf.AlphaChannelError: Refusing to work on images with alpha channel

2022-01-25 12:14:09,995:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 44, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/PIL/Image.py", line 546, in __getattr__
    raise AttributeError(name)
AttributeError: filename

2022-01-25 12:17:27,642:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 44, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/PIL/Image.py", line 546, in __getattr__
    raise AttributeError(name)
AttributeError: filename

2022-01-25 12:19:07,876:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 44, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 2102, in convert
    ) in read_images(rawdata, kwargs["colorspace"], kwargs["first_frame_only"]):
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1483, in read_images
    imgdata, imgformat, default_dpi, colorspace, rawdata
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1189, in get_imgmetadata
    raise AlphaChannelError("Refusing to work on images with alpha channel")
img2pdf.AlphaChannelError: Refusing to work on images with alpha channel

2022-01-25 12:20:34,181:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 43, in check_and_convert_extension
    image = image[:,:,:3]
TypeError: 'PngImageFile' object is not subscriptable

2022-01-25 12:25:40,718:__main__:
Filename: MicrosoftTeams-image (6).png. Error occurred in Digitization.
Traceback (most recent call last):
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/image_segmentation.py", line 313, in digitize_main
    pdf_path = check_and_convert_extension(pdf_path)
  File "/drive/mnt/pradeep/invoiceProduct_solution/apps/multiple_format_to_pdf.py", line 40, in check_and_convert_extension
    pdf_bytes = img2pdf.convert(image.filename)
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 2102, in convert
    ) in read_images(rawdata, kwargs["colorspace"], kwargs["first_frame_only"]):
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1483, in read_images
    imgdata, imgformat, default_dpi, colorspace, rawdata
  File "/data/common/miniconda3/envs/cba_gpu/lib/python3.7/site-packages/img2pdf.py", line 1189, in get_imgmetadata
    raise AlphaChannelError("Refusing to work on images with alpha channel")
img2pdf.AlphaChannelError: Refusing to work on images with alpha channel
